lms/djangoapps/ccx/views.py

```diff
@@ -507,12 +521,19 @@ def ccx_student_management(request, course, ccx=None):
 
     if error_message is None:
         if action == 'add':
-            # by decree, no emails sent to students added this way
-            # by decree, any students added this way are auto_enrolled
-            enroll_email(course_key, email, auto_enroll=True, email_students=False)
+            # check if there are spots left
+            ccx_course_overview = CourseOverview.get_from_id(course_key)
+            if CourseEnrollment.objects.is_course_full(ccx_course_overview):
```

Comment:
@giocalitri  i think you need to update logic of ```ccx_student_management```, 
If ```is_course_full is True``` then there is no point of validate student , just check in start of function if  mode is ```enroll``` and  ```is_course_full is True``` then inform coach that he cannot add student.